src/_locales/en_US/messages.json

```diff
@@ -594,5 +598,13 @@
     "popup_special_page_paragraph": {
         "message": "Privacy Badger doesn't work on special pages like this one. Try browsing somewhere else.",
         "description": "Shown in the popup for special browser pages such as the New Tab page and 'about:' pages."
+    },
+    "widget_control_tab": {
+        "message": "Widgets",
+        "description": "Options page tab heading"
+    },
+    "widget_blocking_desc": {
+        "message": "Privacy Badger automatically replaces widgets with trackers embedded in them. If there are widgets that you'd like to be removed rather than replaced, you can manage that here.",
+        "description": "Description of Privacy Badger widget blocking and replacement"
```

Comment:
Should instead describe where this text can be found (on the options page under the Widget Replacement tab) by translators.

Reply:
Why don't we say something like, Introduction to the Widget Replacement tab on the options page.